app/lib/lets-encrypt.server.ts

```diff
@@ -416,11 +416,18 @@ class LetsEncrypt {
     });
 
     this.#order = await this.#client.finalizeOrder(this.#order, csr);
-    const certificate = await this.#client.getCertificate(this.#order);
+
+    // fullChain is the publicCertificate + the intermediate certificate(s) for Let's Encrypt
```

Comment:
space after `public` before `Certificate`, lower case `c` in `certificate`

Reply:
Weird how I did a typo in the comment. Thanks for catching that